MapboxMobileEvents/MMEAPIClient.h

```diff
@@ -20,6 +20,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)getConfigurationWithCompletionHandler:(nullable void (^)(NSError * _Nullable error, NSData * _Nullable data))completionHandler;
 - (void)reconfigure:(MMEEventsConfiguration *)configuration;
+- (NSError *)statusErrorFromRequest:(nonnull NSURLRequest *)request andHTTPResponse:(nonnull NSHTTPURLResponse *)httpResponse;
```

Comment:
`nonnull` is implicit for all methods in this header because it uses the [`NS_ASSUME_NONNULL_BEGIN/END` macros](https://developer.apple.com/swift/blog/?id=25), so specifying it is redundant and unnecessary.